deploy/helm/sumologic/values.yaml

```diff
@@ -4669,6 +4669,10 @@ opentelemetry-operator:
         cpu: 150m
         memory: 256Mi
 
+  kubeRBACProxy:
+    image:
+      tag: v0.14.4
```

Comment:
Is there any reason for us to change this?

Reply:
Oh, I double checked it and actually don't. I mixed gcr and quay repos